src/ui/components/AppWrapper/AppWrapper.tsx

```diff
@@ -96,6 +96,15 @@ const processMultiSigIcpNotification = async (
   retryInterval = 3000
 ) => {
   try {
+    if (!event.multisigId) {
+      return;
+    }
+    const isMultisigInitiator = await Agent.agent.multiSigs.isMultisigInitiator(
+      event.multisigId
+    );
+    if (isMultisigInitiator) {
```

Comment:
We can do this check in `signifyNotificationService.ts` - only store it if we need to do some user action with it (e.g. its the first multisig/icp for this message)